### PR TITLE
Fix theming and use updated mehtod name

### PR DESCRIPTION
**Pull request by libpitt**
Change log:
1. Resolve modal style in Uploads section
2. Fix call to modal

### CODE REVIEW

**Review by yuanzhou — APPROVED** [review]

**Comment by yuanzhou** [comment]
@maxsibilla I'll wait until you also approve this PR to get things on HuBMAP DEV.

**Review by maxsibilla — APPROVED** [review]
@yuanzhou Yes the modal looks good in this PR

**Pull request merged**
